MethodDecorator.Fody.Tests/MethodDecorator.Fody.Tests.csproj

```diff
@@ -77,9 +77,7 @@
     <Compile Include="Helpers\AssemblyExtensions.cs" />
     <Compile Include="SimpleTestBase.cs" />
     <Compile Include="TestsBase.cs" />
-    <Compile Include="Method.cs">
-      <DependentUpon>TestsBase.cs</DependentUpon>
-    </Compile>
```

Comment:
Not sure if this was intentional but Visual Studio shows TestsBase.cs as a child of Method.cs in the Solution Explorer tree which confused me initially because at first it seemed that TestsBase is not even in the project. Then I found it as a child of Method.cs which was really strange. Is there a reason for having this dependency there?